src/lib/flow/sol-flow-builder.c

```diff
@@ -955,3 +1143,97 @@ sol_flow_builder_export_out_port(struct sol_flow_builder *builder, const char *n
 
     return 0;
 }
+
+static size_t
+get_member_alignment(const struct sol_flow_node_options_member_description *member)
+{
+    struct sol_str_slice t;
+    static const struct sol_str_table alignments[] = {
+        SOL_STR_TABLE_ITEM("boolean", __alignof__(member->defvalue.b)),
```

Comment:
`__alignof__` is a GCC extension. Is it supported by all compilers we're also supporting?